tests/rbd/rbd_snap_clone_imported_image.py

```diff
@@ -0,0 +1,44 @@
+from tests.rbd.rbd_utils import Rbd
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """
+    Test case covered -
+    CEPH-9230 - verification snap and clone on an imported image
+    Pre-requisites :
+    1. Cluster must be up and running with capacity to create pool
+       (At least with 64 pgs)
+    2. We need atleast one client node with ceph-common package,
+       conf and keyring files
+
+    Test Case Flow:
+    1. Create a pool
+    2. Create a dummy file, import it as an image
+    3. Create snapshot of it and protect the snapshot
+    4. Create a clone
+    """
+    log.info("Running snap and clone operations on imported image")
+    rbd = Rbd(**kw)
+    pool = rbd.random_string()
+    image = rbd.random_string()
+    snap = rbd.random_string()
+    dir_name = rbd.random_string()
+    clone = rbd.random_string()
+
+    rbd.exec_cmd(cmd="mkdir {}".format(dir_name))
+    rbd.create_pool(poolname=pool)
+
+    rbd.create_file_to_import(filename="dummy_file")
+    rbd.exec_cmd(cmd="rbd import {} {}/{}".format("dummy_file", pool, image))
+    rbd.exec_cmd(cmd="rbd snap create {}/{}@{}".format(pool, image, snap))
+    rbd.exec_cmd(cmd="rbd snap protect {}/{}@{}".format(pool, image, snap))
+    rbd.exec_cmd(
+        cmd="rbd clone {pool}/{}@{} {pool}/{}".format(image, snap, clone, pool=pool)
+    )
+
+    rbd.clean_up(pools=[pool])
```

Comment:
would it be a good idea to add library module for Rbd to handle all supported operations available in `Rbd()`.